src/clients/c++/request_http.cc

```diff
@@ -1453,6 +1453,10 @@ InferHttpContextImpl::PreRunProcessing(std::shared_ptr<Request>& request)
     curl_easy_setopt(curl, CURLOPT_VERBOSE, 1L);
   }
 
+  const long buffer_byte_size = 16 * 1024 * 1024;
```

Comment:
Is this an empirical value?

Reply:
No. But the maximum that curl allows is 2MB so I just wanted to be bigger than that.